manifests/s/Seewo/EasiNote/5.2.3.5046/Seewo.EasiNote.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.2.12 by Dumplings $debug=QUSU.CRLF.7-4-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Seewo.EasiNote
+PackageVersion: 5.2.3.5046
+InstallerType: nullsoft
+Scope: machine
+UpgradeBehavior: install
+Protocols:
+- easinote
+FileExtensions:
+- enb
+- enbx
+- enpx
+ReleaseDate: 2023-12-20
+Installers:
+- Architecture: x86
+  InstallerUrl: https://cstore-pub-cos-seewo-report-tx.seewo.com/seewo-report_uwixvoojhhulnhlwhlmhuxnowyhhihhh?sign=q-sign-algorithm%3Dsha1%26q-ak%3DAKIDJUXMxJRLzPaeMp20jDSTFl23pLcdPwDF%26q-sign-time%3D1703040686%3B2018400686%26q-key-time%3D1703040686%3B2018400686%26q-header-list%3Dhost%26q-url-param-list%3Dresponse-content-disposition%26q-signature%3D66e79c05b4bc1bf62e6231b874b33dc44294a4d5&response-content-disposition=attachment%3Bfilename%3D%22EasiNoteSetup_5.2.3.5046_seewo.exe%22%3Bfilename%2A%3Dutf-8%27%27EasiNoteSetup_5.2.3.5046_seewo.exe

```

Comment:
This InstallerUrl seems to have a time-based component.

Reply:
Maybe not. This URL comes from their API without any redirections. You can still access to the early versions of installers.